core/resource/handler/v1beta1/resource.go

```diff
@@ -197,7 +197,9 @@ func (rh ResourceHandler) CreateResource(ctx context.Context, req *pb.CreateReso
 		return nil, errors.GRPCErr(err, "failed to create resource "+res.FullName())
 	}
 
-	return &pb.CreateResourceResponse{}, nil
+	return &pb.CreateResourceResponse{
+		Success: true,
```

Comment:
Please do not sent `Success` field. We should not be using this field. This field is useless.